pkg/ccl/backupccl/backup_planning.go

```diff
@@ -399,6 +353,168 @@ func checkPrivilegesForBackup(
 	return nil
 }
 
+// privilegesDeprecationNotice returns a notice that outlines the deprecation of
+// the existing privilege model for backups, and the steps to take to adopt the
+// new privilege model, based on the type of backup being run.
+func privilegesDeprecationNotice(
+	p sql.PlanHookState, backupStmt *annotatedBackupStatement, targetDescs []catalog.Descriptor,
+) string {
+	if backupStmt.Targets == nil {
+		return ""
+	}
+
+	// If a user is running `BACKUP DATABASE` buffer all the databases that will
+	// require the `BACKUP` privilege.
+	var notice string
+	preamble := "The existing privileges required for backup are being deprecated " +
+		"in favour of a fine-grained privilege model explained here <link>. In 22.2 we will support " +
+		"both the old and new privilege requirements, but in 23.1 to run"
```

Comment:
Perhaps we can shorten this:
```suggestion
		"in favour of a fine-grained privilege model explained here <link>. In a future release, to run"
```

Reply:
done.